static/src/javascripts/projects/commercial/modules/liveblog-adverts.ts

```diff
@@ -92,12 +93,17 @@ const getSpaceFillerRules = (
 	};
 };
 
+// On mobile devices, the first inline slot is called "top-above-nav", despite it
```

Comment:
I missed this in the review but I assume we want this change in the commercial repo and not in the deprecated code in Frontend :)